Npm free asset without npm

f1amy opened the pull request:
Using npm-asset without need in npm.
@rmrevin, can you look at my code and see if this can be implemented?

rmrevin commented on composer.json:
```diff
@@ -24,7 +24,8 @@
     "php": ">=5.4.0",
-    "yiisoft/yii2": "^2.0.0"
+    "yiisoft/yii2": "^2.0.0",
+    "npm-asset/fortawesome--fontawesome-free": "~5.7.1"
```
Not sure if unconditional installation of the package is appropriate. Many people use the pro version, they do not need free.

f1amy replied:
Ok I'll remove it and update docs

f1amy commented:
I updated PR, removed unconditional installation and updated docs.
Please check [this](https://github.com/hiqdev/asset-packagist/issues/97), so when they fix the search, update README where
```
composer require "npm-asset/fortawesome--fontawesome-free:~5.7.1"
```
to
```
composer require "npm-asset/@fortawesome/fontawesome-free:~5.7.1"
```

rmrevin commented:
Thanks.
